corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -762,6 +765,12 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             }
         },
 
+        updateSubmitButtonStatus: function (disabled) {
+            if (this.options.sidebarEnabled) {
+                sessionStorage.submitDisabled = disabled;
```

Comment:
I feel like this would be clearer with slightly different naming. Maybe along the lines of:
```suggestion
        updateSubmitButtonStatus: function (shouldDisable) {
            if (this.options.sidebarEnabled) {
                sessionStorage.disableSubmit = shouldDisable;
```

My thought being that `submitDisabled` sounds like it's based on the current state of the submit button, rather than a flag used to set that state.